### ISSUE TITLE
PDF output does not render keyboard shortcut graphics

### ISSUE THREAD

**Issue by r0ckarong** [asciidoctor/asciidoctor-pdf#1004]
I'm wondering if this will be added at some point. It's a very useful feature in the HTML output but in the PDF it just renders the key combination as the string you entered in the macro.

`kbd:[Alt+F4]` does not become the Alt + F4 buttons but just says (in literal type) `Alt+F4`. While this technically does not lose information it's a shame.

**Comment by habamax** [comment]
Well I have themed it that way:
![image](https://user-images.githubusercontent.com/234774/61027769-a8590c80-a3bf-11e9-9c39-08693d0ad404.png)

Is it what you are looking for?

If so, add to you theme:
```
literal:
  font_color: #c0006a
  font_family: Literal
  font_size: $base_font_size
  background_color: #eeeeee
  border_color: #eeeeee
  border_radius: 1
  border_width: 0
  border_offset: 1
```

The drawback is that it is `literal` meaning that all your inline code will be styled that way. Which for me is ok. :)

**Comment by mojavelinux** [comment]
I believe this is a duplicate of #313. The necessary groundwork is now in place, to we're looking to add the configuration in beta.2.

**Comment by habamax** [comment]
I believe this is fixed now?
I have tried recent master and was able to confiure `key` block to have distinct bordered shortcuts.

**Comment by mojavelinux** [comment]
Absolutely correct. We addressed this in #313 (though technically this issue was more descriptive for this case). I'll mark it as fixed in beta.2.
